src/applications/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -2119,9 +2119,7 @@ export const getErrorStatus = response => {
       Sentry.captureException(response);
     });
   }
-  return response.errors && response.errors.length
-    ? response.errors[0].status
-    : UNKNOWN_STATUS;
+  return response?.errors?.[0]?.status ?? UNKNOWN_STATUS;
```

Comment:
Made this a little bit safer. Now when an undefined `response` is passed it, it returns UNKNOW_STATUS.